packages/core/src/browser/status-bar/status-bar.tsx

```diff
@@ -103,12 +105,24 @@ export class StatusBarImpl extends ReactWidget implements StatusBar {
         this.internalSetBackgroundColor(color);
     }
 
+    async setForegroundColor(color?: string): Promise<void> {
+        await this.ready;
+        this.internalSetForegroundColor(color);
+    }
+
     protected internalSetBackgroundColor(color?: string): void {
         this.backgroundColor = color;
         // tslint:disable-next-line:no-null-keyword
         this.node.style.backgroundColor = this.backgroundColor ? this.backgroundColor : null;
     }
 
+    protected internalSetForegroundColor(color?: string): void {
+        this.foregroundColor = color;
+        this.entries.forEach(entry => {
+            entry.color = this.foregroundColor ? this.foregroundColor : undefined;
+        });
```

Comment:
@vince-fugnitto Foreground is black for most of the field except for the "ln and column" of an editor which are still white when offline

Reply:
@lmcbout do you mean this?
<img width="974" alt="screen shot 2018-11-16 at 3 38 59 pm" src="https://user-images.githubusercontent.com/40359487/48646116-d3689900-e9b5-11e8-9b7d-4413c3fdf2df.png">